app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -109,6 +112,30 @@ class ExplorationActivityPresenter @Inject constructor(
     }
   }
 
+  /** Action for onOptionsItemSelected */
+  fun clickOnOverFlowMenuItems(item: MenuItem?): Boolean {
```

Comment:
Rename this to : `handleOnOptionsItemSelected `

Reply:
Done